src/js/features/network/saga.js

```diff
@@ -17,7 +17,8 @@ export function *doInit() {
   try {
     yield call(onReady);
     const networkId = yield call(web3.eth.net.getId);
-    yield put({type: INIT_SUCCEEDED, networkId});
+    const gasPrice = yield web3.eth.getGasPrice();
+    yield put({type: INIT_SUCCEEDED, networkId, gasPrice});
```

Comment:
Here, you can just send the GET_PRICE action so that the other function will do it for you.
```
yield put({type: GET_GAS_PRICE});
```
You can even put it in a `all` to make it parallel.